src/triage/component/postmodeling/contrast/model_evaluator.py

```diff
@@ -0,0 +1,1199 @@
+
+    @property
+    def hyperparameters(self):
+        return self.metadata['hyperparameters']
+
+    @property
+    def model_hash(self):
+        return self.metadata['model_hash']
+
+    @property
+    def train_matrix_uuid(self):
+        return self.metadata['train_matrix_uuid']
+
+    @property
+    def pred_matrix_uuid(self):
+        return self.metadata['matrix_uuid']
+
+    @property
+    def as_of_date(self):
+        return self.metadata['as_of_date']
+
+    def __repr__(self):
+        return (
+        str({'model_id': self.model_id,
+             'model_group': self.model_group_id,
+             'model_type': self.model_type,
+             'as_of_date': self.as_of_date.strftime("%B %d, %Y"),
+             'model_hyperparameters': self.hyperparameters})
+        )
+
+    @cachedproperty
+    def predictions(self):
+        preds = pd.read_sql(
+            f'''
+            SELECT model_id,
+                   entity_id,
+                   as_of_date,
+                   score,
+                   label_value,
+                   COALESCE(rank_abs, RANK() OVER(ORDER BY score DESC)) AS rank_abs,
+                   COALESCE(rank_pct, percent_rank()
+                   OVER(ORDER BY score DESC)) * 100 as rank_pct,
+                   test_label_timespan
+            FROM test_results.predictions
+            WHERE model_id = {self.model_id}
+            AND label_value IS NOT NULL
+            ''', con=self.engine)
+
+        return preds
+
+    def _feature_importance_slr(self, path=None):
+        '''
+        Calculate feature importances for ScaledLogisticRegression
+
+        Since triage do not calculate the feature importances of
+        ScaledLogisticRegression by default, this function will call the model
+        object and calculate a feature importance proxy using the coefficients
+        of the regression.
+
+        Arguments:
+            - path (str): triage's project path, or a path to find the model
+            objects
+
+        Return:
+            - Dataframe with feature_importances
+        '''
+
+        storage = ProjectStorage(path)
+        model_obj = ModelStorageEngine(storage).load(self.model_hash)
+        coefficients = np.abs(1 - np.exp(model_obj.coef_.squeeze()))
+
+        self.preds_matrix(path)
+        test_matrix = self.preds_matrix(path)
+        feature_names = [x for x in test_matrix.column.tolist() 
+                         if x not in self.predictions.column.tolist()]
+
+        raw_importances = pd.DataFrame(
+            {'feature': feature_names,
+             'model_id':  test_matrix['model_id'],
+             'feature_importance': coefficients,
+             'feature_group': [x.split('_entity_id')[0] for x in feature_names]
+            }
+        )
+        raw_importances['rank_abs'] = raw_importances['feature_importance'].\
+                                      rank(method='max')
+
+        return raw_importances
+
+
+    def feature_importances(self, path=None):
```

Comment:
(ditto on `path` being `None`)

Reply:
Hmmmm this one is tricky. 

If the model has a triage's feature importance calculated, then I will not need any path. On the other hand, if the feature importance is not available, then I'd need the `feature_importance_slr` function to kick in and calculate those. 

[This in actually a temporary patch for a triage problem that now is causing me a constant headache in all this post modeling process. Can you help me solve the #509 @thcrock @nanounanue , I know is easy, but I need you help to figure it out fast].

Reply:
I just create a PR [#587] solving this problem. Once we merge it, I can get rid of all the feature_importances hacking.